src/main/java/com/internet/shop/Application.java

```diff
@@ -0,0 +1,26 @@
+package com.internet.shop;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.model.Product;
+import com.internet.shop.service.ProductService;
+
+public class Application {
+    private static Injector injector = Injector.getInstance("com.internet.shop");
+
+    public static void main(String[] args) {
+        ProductService productService = (ProductService) injector.getInstance(ProductService.class);
+
+        productService.create(new Product("Iphone 10", 1_200));
+        productService.create(new Product("Iphone 11", 1_400));
+        productService.create(new Product("Iphone X", 1_600));
+
+        System.out.println(productService.getAll());
+        productService.deleteById(2L);
```

Comment:
let's save our products to variables and use their ids through `someProduct.getId()` when needed